crew/master/tornado/__init__.py

```diff
@@ -1 +1 @@
-from .client import Client
+from crew.master.tornado.client import Client
```

Comment:
I use python 3.4 and fail the same place that @degibenz fails, so that I've replaced all the relative imports with absolute ones.